Userland/Applications/Browser/InspectorWidget.cpp

```diff
@@ -151,6 +160,36 @@ void InspectorWidget::load_style_json(String specified_values_json, String compu
     m_custom_properties_table_view->set_model(Web::StylePropertiesModel::create(m_inspected_node_custom_properties_json.value().view()));
 }
 
+void InspectorWidget::update_node_box_model(Optional<String> node_box_sizing_json)
+{
+    if (!node_box_sizing_json.has_value()) {
+        return;
+    }
+    auto json_or_error = JsonValue::from_string(node_box_sizing_json.value());
+    if (json_or_error.is_error() || !json_or_error.value().is_object()) {
+        return;
+    }
+    auto json_value = json_or_error.release_value();
+    const auto& json_object = json_value.as_object();
+
+    m_node_box_sizing.margin.top = static_cast<float>(json_object.get("margin_top").to_double());
+    m_node_box_sizing.margin.right = static_cast<float>(json_object.get("margin_right").to_double());
+    m_node_box_sizing.margin.bottom = static_cast<float>(json_object.get("margin_bottom").to_double());
+    m_node_box_sizing.margin.left = static_cast<float>(json_object.get("margin_left").to_double());
+    m_node_box_sizing.padding.top = static_cast<float>(json_object.get("padding_top").to_double());
+    m_node_box_sizing.padding.right = static_cast<float>(json_object.get("padding_right").to_double());
+    m_node_box_sizing.padding.bottom = static_cast<float>(json_object.get("padding_bottom").to_double());
+    m_node_box_sizing.padding.left = static_cast<float>(json_object.get("padding_left").to_double());
+    m_node_box_sizing.border.top = static_cast<float>(json_object.get("border_top").to_double());
+    m_node_box_sizing.border.right = static_cast<float>(json_object.get("border_right").to_double());
+    m_node_box_sizing.border.bottom = static_cast<float>(json_object.get("border_bottom").to_double());
+    m_node_box_sizing.border.left = static_cast<float>(json_object.get("border_left").to_double());
```

Comment:
couldn't you just have added a to_float?

Reply:
Fixed and force pushed.